src/skills/hashtags/mod.ts

```diff
@@ -0,0 +1,50 @@
+import { SkillModule } from "/src/platform/skillModules/types/SkillModule.ts";
+import { SkillListener } from "/src/platform/skillModules/types/SkillListener.ts";
+import { hashtagMentionListener } from "./listeners/hashtagMention.ts";
+import { cmdJoin } from "./commands/cmdJoin.ts";
+import { cmdLeave } from "./commands/cmdLeave.ts";
+import { getInitialHashtagChannelSessionData } from "./sessionData/getInitialTagChannelSessionData.ts";
+import { SessionData } from "../../context/mod.ts";
+
+export const name: SkillModule["name"] = "hashtags";
+export const initializers: SkillModule["initializers"] = [];
+export const middlewares: SkillModule["middlewares"] = [];
+export const commands: SkillModule["commands"] = [
+  {
+    command: "join_hashtag",
+    aliases: [],
+    description:
+      "Join hashtag channel and get notified. Example: /join_hashtag #games",
+    handler: cmdJoin,
+  },
+  {
+    command: "leave_hashtag",
+    aliases: [],
+    description: "Leave a hashtag channel. Example: /leave_hashtag #games",
+    handler: cmdLeave,
+  },
+];
+
+export const sessionDataInitializers: SkillModule["sessionDataInitializers"] = [
+  getInitialHashtagChannelSessionData,
+];
+
+export const listeners: SkillModule["listeners"] = [
+  <SkillListener<"message:text">> {
+    event: "message:text",
+    description:
+      "Listens to a hashtag and mentions the people registered on it.",
+    handler: hashtagMentionListener,
+  },
+];
+
+export const inlineQueryListeners: SkillModule["listeners"] = [];
+
+export const migrations: SkillModule["migrations"] = {
+  1: function addHashtagChannelsToSession(old: SessionData): SessionData {
```

Comment:
this will bite me in the ass_in the future

better figure another way to merge these migrations so their execution order is consistent across domains